### ISSUE TITLE
Can't wait for it

### ISSUE THREAD

**Issue by r-k-o** [congdm/AyaCompiler#13]
hi,
I am glad you are back and work on the compiler.
Maybe you can emit more x64 objects like libs and allow for easier module imports of large windows headers.

**Comment by congdm** [comment]
Yes, I'm thinking about it, but the first priority is to restructure the modules, especially the base because it's too platform dependent. For windows, this time I will make sure the DLL loading code and x64 SEH exception information (for debugger) are working properly.
